lib/hanami/assets/helpers.rb

```diff
@@ -125,9 +131,11 @@ def self.included(base)
       #   <%= javascript 'application' %>
       #
       #   # <script src="https://assets.bookshelf.org/assets/application-28a6b886de2372ee3922fcaf3f78f2d8.js" type="text/javascript"></script>
-      def javascript(*sources)
+      def javascript(*sources, **options)
```

Comment:
What do you think about a default hash `async: true` instead?

And why do you implement the option here at this high level than in the lower level of `script`?